### DISCUSSION TITLE
Templating ini-file with duplicate sections?

### DISCUSSION THREAD

**Discussion by strowi** [grafana/tanka#831] [Q&A]
Hi,

i am currently trying to template a configMap (for influxdb-relay)  that is basically an ini-file with duplicate sections:

```
# InfluxDB Backend InfluxDB01
[[influxdb]]
  name = "influxdb01"
  location = "http://127.0.0.1:8086/"
  timeout = "10s"

# InfluxDB Backend InfluxDB02
[[influxdb]]
  name = "influxdb02"
  location = "http://127.0.0.1:8087/"
  timeout = "10s"
...
```

by looping of a list of influxDB-instances. The problem is `std.manifestIni` won't allow this. Does anyone have an idea how to do this?

**Comment by Duologic** [comment]
Can you show what you've tried already?

**Comment by frimik** [comment]
The configuration is actually toml, not ini.

look at `std.manifestTomlEx(toml, indent)`

Available since version 0.18.0.

**Comment by strowi** [comment]
Thx will try that. The example on https://jsonnet.org/ref/stdlib.html is a bit hard to read. Thought duplicate-keys would be a problem there too.
